src/components/dashboard/Relaying/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { WidgetBody, WidgetContainer } from '@/components/dashboard/styled'
+import { Box, Card, Divider, Grid, Link, Skeleton, SvgIcon, Typography } from '@mui/material'
+import RelayerIcon from '@/public/images/common/relayer.svg'
+import css from './styles.module.css'
+import useRemainingRelays from '@/hooks/useRemainingRelays'
+import useChains, { useCurrentChain } from '@/hooks/useChains'
+import { FEATURES, hasFeature } from '@/utils/chains'
+import ImageFallback from '@/components/common/ImageFallback'
+
+const FallbackChainIcon = ({ color }: { color: string }) => (
+  <div
+    style={{
+      width: '16px',
+      height: '16px',
+      backgroundColor: color,
+      borderRadius: '50%',
+    }}
+  />
+)
+
+const Relaying = () => {
+  const [remainingRelays] = useRemainingRelays()
+  const { configs } = useChains()
+  const currentChain = useCurrentChain()
+
+  return (
+    <WidgetContainer>
+      <Typography component="h2" variant="subtitle1" fontWeight={700} mb={2}>
+        Gasless transactions
+      </Typography>
+
+      <WidgetBody>
+        <Card sx={{ padding: 3, height: 'inherit' }}>
+          <Grid container mb={3}>
+            <Grid item xs={12} sm={2}>
+              <SvgIcon component={RelayerIcon} sx={{ width: 'auto', height: '60px' }} inheritViewBox />
+            </Grid>
+            <Grid item xs={10}>
+              <Box display="flex" alignItems="center">
+                <Typography variant="h6" fontWeight={700}>
+                  Introducing Relayer
+                </Typography>
+                <Box className={css.inlineChip} sx={{ backgroundColor: 'secondary.light' }}>
+                  New
+                </Box>
+              </Box>
+              <Typography variant="body2" sx={{ display: 'inline' }}>
+                Benefit from gasless experience powered by Gelato and Safe.{' '}
+              </Typography>
+              <Link href="#" color="primary.main" fontWeight="bold">
```

Comment:
I assume this will be a help article? If so, this should be an `ExternalLink`. I'd also suggest adding a "TODO" comment here so we don't forget.